apps/compute-cf.yml.j2

```diff
@@ -1,5 +1,26 @@
 AWSTemplateFormatVersion: 2010-09-09
 
+
+{%- set compute_envs = [] -%}
+{%- set compute_env_names = [] -%}
+{%- for job_type, job_spec in job_types.items() -%}
+  {%- for task in job_spec['tasks'] -%}
+    {% set env = task['compute_environment'] %}
+    {% if 'name' in env and env['name'] != 'Default' and env['name'] not in compute_env_names %}
```

Comment:
Does the `env['name'] not in compute_env_names` condition allow multiple tasks to define compute environments with the same name, and uses whichever one is defined first, and then the remaining compute environments get silently ignored? If that's the current behavior, then I'd prefer to instead raise an exception if multiple compute environments have the same name.